src/ameba/config.cr

```diff
@@ -138,7 +139,7 @@ class Ameba::Config
   # ```
   #
   def update_rule(name, enabled = true, excluded = nil)
-    index = @rules.index { |r| r.name == name }
+    index = @rules.index(&.name.== name)
```

Comment:
TBH i find the previous version mode readable

Reply:
Ok, I've reverted this change.